incubator/gogs/values.yaml

```diff
@@ -0,0 +1,213 @@
+---
+## Override the name of the Chart.
+##
+# nameOverride:
+
+## Kubernetes configuration
+## For minikube, set this to NodePort, elsewhere use LoadBalancer
+##
+serviceType: NodePort
+
+replicaCount: 1
+
+image:
+  repository: gogs/gogs
+  tag: 0.9.141
+  pullPolicy: IfNotPresent
+
+service:
+  ## Override the components name (defaults to service).
+  ##
+  # nameOverride:
+
+  ## HTTP listen port.
+  ## ref: https://gogs.io/docs/advanced/configuration_cheat_sheet
+  ##
+  httpPort: 80
+
+  ## SSH listen port.
+  ## ref: https://gogs.io/docs/advanced/configuration_cheat_sheet
+  ##
+  sshPort: 22
+
+  ## Gogs configuration values
+  ## ref: https://gogs.io/docs/advanced/configuration_cheat_sheet
+  ##
+  gogs:
+
+    ## Application name, can be your company or team name.
+    ##
+    appName: 'Gogs: Go Git Service'
```

Comment:
Should probably simplify to just **Gogs**, official repo has removed verbose part, but not a big deal..